composer.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "spryker-eco/unzer-gui",
+    "type": "library",
+    "description": "UnzerGui module",
+    "license": "MIT",
+    "require": {
+        "php": ">=7.4",
+        "spryker-eco/unzer": "dev-feature/cc-14610/dev-unzer-psp as 1.0",
+        "spryker/gui": "^3.33.0",
+        "spryker/kernel": "^3.30.0",
+        "spryker/merchant": "^3.0.0",
+        "spryker/merchant-gui": "^3.8.0",
```

Comment:
looks like a wrong dependency 
probably `Spryker\Zed\MerchantGui\Communication\Exception\MissingStoreRelationFormTypePluginException` is the reason

Reply:
Fixed.